src/coreclr/interpreter/compiler.cpp

```diff
@@ -3710,6 +3711,42 @@ int InterpCompiler::GenerateCode(CORINFO_METHOD_INFO* methodInfo)
                 break;
             }
 
+            case CEE_LDTOKEN:
+            {
+                AddIns(INTOP_LDTOKEN);
+
+                CORINFO_RESOLVED_TOKEN resolvedToken = { 0 };
+                resolvedToken.tokenScope = m_compScopeHnd;
+                resolvedToken.tokenContext = METHOD_BEING_COMPILED_CONTEXT();
+                resolvedToken.token = getU4LittleEndian(m_ip + 1);
+                resolvedToken.tokenType = CORINFO_TOKENKIND_Ldtoken;
```

Comment:
It would be nicer to use the `ResolveToken` method that we already have for consistency